client/packages/system/src/Patient/ListView/ListView.tsx

```diff
@@ -120,10 +121,12 @@ export const PatientListView = () => {
         enableColumnFilter: true,
         size: 120,
         filterVariant: 'select',
-        filterSelectOptions: Object.values(GenderTypeNode).map(status => ({
-          value: status,
-          label: t(getGenderTranslationKey(status)),
-        })),
+        filterSelectOptions: Object.values(GenderTypeNode)
```

Comment:
Can you filter this out by pref gender please thanks, then wouldn't need above (:

Reply:
Thats a cleaner solution. Please see changes: https://github.com/msupply-foundation/open-msupply/pull/10002/commits/9c94eb931df83c6f3730275f01e1f510a5cda4fe

Reply:
`genderOptions.map`...   then you don't need to filter the gender type node 👀